Observer/CaptureObserver.php

```diff
@@ -262,24 +263,29 @@ private function shippingTrackingInfo($invoice)
      * @param $orderObject
      * @param $payment
      * @param $storeCode
+     * @param $paymentType
      *
      * @throws ResponseHeaderException
      */
-    private function sendInvoiceRequest($invoice, $orderLines, $orderObject, $payment, $storeCode)
+    private function sendInvoiceRequest($paymentType, $invoice, $orderLines, $orderObject, $payment, $storeCode)
     {
-        $api = new CaptureReservation($this->systemConfig->getAuth($storeCode));
-        if ($invoice->getTransactionId()) {
-            $api->setInvoiceNumber($invoice->getTransactionId());
-        }
-        $api->setAmount((float)number_format($invoice->getGrandTotal(), 2, '.', ''));
-        $api->setOrderLines($orderLines);
-        $shippingTrackingInfo = $this->shippingTrackingInfo($invoice);
-        /*send shipping tracking info if exists*/
-        if (!empty($shippingTrackingInfo)) {
+        $grandTotal = (float)$invoice->getGrandTotal();
```

Comment:
What type is `getGrandTotal` returning?

Reply:
It's returning string

Reply:
on, round should do an implicit cast, but doing an explicit one is also fine